mlflow/onnx.py

```diff
@@ -255,9 +255,9 @@ def __init__(self, path, providers=None):
             if intra_op_num_threads:
                 sess_options.intra_op_num_threads = intra_op_num_threads
             if execution_mode:
-                if execution_mode == 0:
+                if execution_mode.upper() == "SEQUENTIAL":
```

Comment:
We should document that when logging model, we should use "SEQUENTIAL" or "PARALLEL" as session option value.